ui/components/multichain/create-account/create-account.js

```diff
@@ -86,6 +86,7 @@ export const CreateAccount = ({ onActionComplete }) => {
     <Box as="form" onSubmit={onSubmit}>
       <FormTextField
         autoFocus
+        id="account-name"
```

Comment:
this is to remove console warnings which can be seen when running e2e tests now as well